modules/face/include/opencv2/face/face_alignment.hpp

```diff
@@ -0,0 +1,306 @@
+/*
+By downloading, copying, installing or using the software you agree to this
```

Comment:
There is a shortened version of the header in all new files. Its only 3 rules pointing to the correct license file. Might want to select that one!